src/app/static/src/app/components/modelOutput/ModelOutput.vue

```diff
@@ -1,11 +1,51 @@
 <template>
     <div>
-        <!-- TODO:PC -->
+        <ul class="nav nav-tabs">
+            <li v-for="plotName of plotNames" :key="plotName">
+                <a class="nav-link"
+                   :class="selectedPlot === plotName ? 'active': ''"
+                   v-translate="plotName"
+                   @click="switchTab(plotName)"></a>
+            </li>
+        </ul>
+        <div class="mt-2">
+            <PlotControlSet :active-plot="selectedPlot"/>
```

Comment:
I think we've stuck to kebab case for component names in hint, same in other files